### DISCUSSION TITLE
How to sync app brightness slider with monitor brightness menu?

### DISCUSSION THREAD

**Discussion by k-matti** [MonitorControl/MonitorControl#1693] [Q&A] [answered]
When I set, for example, brightness at 75% in the app, I see 59 in the monitor menu. How to make these two values in sync?
When I set 75 in the monitor menu, the app still shows the old value.
My monitor is Dell S2722QC connected via USB-C

**Comment by waydabber** [comment]
The app by default uses software dimming in the 0-50% range and DDC control in the 50-100% range. You can change this and the switchover point in settings (the latter in advanced settings).

**Reply by k-matti** [reply]
Which setting should I change to make the app reflect monitor brightness setting and vice versa?

**Reply by waydabber** [reply]
MonitorControl won't update the brightness periodically from the display. If the display properly supports DDC read for brightness, you can configure BetterDisplay (my other app with much more features than MonitorControl) to periodically update the in-app brightness value based on the display's current settings.

**Reply by k-matti** [reply]
Ok, so how set MonitorControl to properly set brightness in one way (app → monitor)? I have found that unchecking the second checkbox on the first tab make difference. Is it correct option?

**Reply by waydabber** [reply, marked as answer]
Yes, disable combined brightness.
